.github/workflows/tests.yml

```diff
@@ -112,6 +112,32 @@ jobs:
         run: |
           asv check -v -E existing
 
+  test_skimage_linux_free_threaded:
+    name: linux-cp313t-default
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout scikit-image
+        uses: actions/checkout@v4
+
+      # TODO: replace with setup-python when there is support
+      - uses: deadsnakes/action@e704e592227156e253e24c7f7ccf2bfe62ef58d4 # v3.1.0
+        with:
+          python-version: "3.13"
+          nogil: true
+
+      - name: Install nightly dependencies
+        run: |
+          set -ex
+          source tools/github/before_install.sh
+          pip install -vv --no-build-isolation .
+
+      - name: Run tests
+        env:
+          PYTHON_GIL: 0
+        run: |
+          source tools/github/script.sh
```

Comment:
This was just discussed in a BoF at SciPy. As I understood, `PYTHON_GIL: 0` will disable free-threading even if the binaries where build with support for it. So I am wondering how this is testing then against free-threaded? Is this just meant to test against the free-threaded build but not with free-threading enabled?

Reply:
Will disable the GIL, presumably, i.e. the opposite!

Reply:
Oh, should have realized. So the free-threaded build from deadsnakes needs this flag in addition? Not including this would just run with the GIL?

Reply:
When all of scikit-image's dependencies are updated to declare they support being imported without the GIL and scikit-image itself is updated this can be removed. The ability to do that in Cython code was merged yesterday so we weren't able to do that for many projects until literally today. It will probably take a week or two to get all the dependencies.

Reply:
See https://github.com/numpy/numpy/pull/26950 for the relevant Cython build changes in NumPy.